### PR TITLE
Adding support for PPC64LE architecture

### PR DESCRIPTION
**Pull request by ezeeyahoo**
Hi,

I have made some changes aiming to add support for PPC64LE architecture, request you to please review.

Thanks

### CODE REVIEW

**Comment by ezeeyahoo** [comment]
Please let me know if there is any blocker in merging this PR.

File: .travis.yml [hunk 1]
```diff
@@ -71,6 +85,11 @@ jobs:
             repo: gridcf/gct
           skip_cleanup: true
           script: bash travis-ci/upload_source_tarballs.sh gridcf
-
-
+        - provider: script
```
Comment (matyasselmeci) [inline .travis.yml:88]: This section looks like a duplicate.

**Comment by matyasselmeci** [comment]
We don't actually deploy any binaries built with the CI, only source tarballs and RPMs. We rely on downstream distributions like EPEL and SuSE to produce binaries. EPEL already produces binary RPMs for PPC64LE -- see https://koji.fedoraproject.org/koji/buildinfo?buildID=1368168 for an example. So this PR doesn't have much of an effect.

**Comment by fscheiner** [comment]
@ezeeyahoo 
Thanks for your PR but as it doesn't really have an effect because our package builds happen downstream at distributions I close this PR. Looking forward to future PRs from you.

**Comment by ezeeyahoo** [comment]
Yes I know and understand, this test is just to make sure that continuous integration of 'head of the tree' is working with ppc architecture. In case of any query, tagging @gerrith3.

**Comment by gerrith3** [comment]
@fscheiner the goal of this is to ensure that top of tree runs on current Power platforms, aka ppc64le.  We have historically had a lot of grid users using/building this indepedently, this just ensure that when people pull the tree they can be confident that it works on Power.  We also do this to reduce the amount of testing/integration needed when building downstream distros, but obviously anyone that wants a more current release than what is included in their distro will download and build from here.

**Comment by brianhlin** [comment]
@ezeeyahoo @gerrith3 one thing to note here is that this relies on Travis CI, which may have to go away sooner rather than later because of their new pricing model. Unfortunately, PowerPC may not be readily available on other CI alternatives (e.g. [GitHub Actions](https://github.com/actions/runner/issues/296)).

**Comment by gerrith3** [comment]
@brianhlin Travis isn't going anywhere for the open source community.  Their messaging was a little off, but you still get 2000 minutes initially from them (which is the limit on CircleCI and GHA btw) but you can ask for as many credits as you need via support.travis-ci.com.  Also, we may be shifting to ppc64le and s390x not actually consuming any credits (my team provides all the hardware for Travis CI at no hardware cost to them).  We partner with them on delivering and improving the service for the open source communities.

**Comment by gerrith3** [comment]
@brianhlin oh, and sorry for two msgs but we are working on getting ppc64le support for a GitHub runner configuration on GHA and CircleCI but those may take a while to complete.

**Comment by brianhlin** [comment]
@gerrith3 I'm not aware of any GHA usage limits for public repositories (see the public vs private repo distinction here https://github.com/features/actions). But I'm glad to hear that there may be PowerPC GHA support on the horizon!

As for Travis, the GCT builds take ~2.5 hours on average so we'll blow through our 2k minutes after about 13 runs -- have you had experience with requesting open source credits from Travis? Since the GridCF doesn't have any dedicated staff, I'm hesitant to rely on a regular, back and forth process to ensure that our CI continues to run. Though if Travis announce that PowerPC usage is free, then that definitely seems like something we could use and we can revisit this PR.

**Comment by nightlark** [comment]
I recently tried to get some open source credits from Travis CI, it took a few weeks to get the initial response about what sounds like a recurring grant of credits. The end result was a one time 60k credits added to my account (rather than the one I'd asked about), which was much less than the request (macOS builds use 5x and Windows builds use 2x the number of credits that Linux builds use). It sounds like there is some form of recurring credit grant

Here's the info they sent about the open source subscription, though from Travis support community posts it sounds like the program might be on hold for an unknown length of time.
```
Thanks for contacting Travis-CI Support! I’d love to help! We offer an Open Source Subscription for free to all non-commercial open-source projects. To qualify for an Open Source subscription, the project must meet the following requirements:
•	You are a project lead or regular committer (latest commit in the last month)
•	Project must be at least 3 months old and is in active development (with regular commits and activity)
•	Project meets the OSD specification
•	Project must not be sponsored by a commercial company or organization (monetary or with employees paid to work on the project)
•	Project can not provide commercial services or distribute paid versions of the software
Sounds like you and your project? We’d be very happy to support you. If your project does not match these requirements or you have questions, feel free to ask!
```

**Comment by gerrith3** [comment]
@brianhlin yes we have a lot of experience with working with travis on getting credits.  I will have to check on the recurring model, I know they are adapting some of this as they go, so I'll need to ask the latest on that.

@nightlark yeah - they were totally swamped after the announcement and their support team was also working on the convergance of their .org and .com sites - and they are still a very small organization.  Things should start getting better as they work through the backlog on credit requests.  As far as the credit cost of MacOS and Windows, I don't deal with that directly but I believe they pay for those VMs themselves for testing; ppc64le and s390x are donated by IBM currently.  Hence the disparity in credits and overall costs.

**Pull request closed without merging**